### PR TITLE
Cut: Remove deprecated deploy_to_cv role

### PR DESCRIPTION
**Pull request by gmuloc**
## Change Summary

Role was deprecated in 4.x and planned for removal in 5.x

## Component(s) name

`arista.avd.deploy_to_cv`

## Proposed changes

Removing the role (there is no process for this in Ansible, only for plugins)

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md [hunk 1]
```diff
@@ -131,6 +131,14 @@ The following ansible plugins have been removed from the `arista.avd` Ansible co
 | Action plugin/Module | `arista.avd.yaml_templates_to_facts` | The internal `arista.avd.yaml_templates_to_facts` action plugin is no longer used by AVD. The plugin is released as open source, so it can be copied and reused according to the license and copyright. |
 | Action plugin/Module | `arista.avd._emit_warning` | The internal `arista.avd._emit_warning` action plugin is no longer used by AVD. The plugin is released as open source, so it can be copied and reused according to the license and copyright. |
 
+## Removal of Ansible roles
+
+The following deprecared ansible roles have been removed from the `arista.avd` Ansible collection in v5.0.0.
```
Comment (carlbuchmann) [inline ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]:
```suggestion
The following deprecated Ansible roles have been removed from the `arista.avd` Ansible collection in v5.0.0.
```

File: ansible_collections/arista/avd/docs/release-notes/5.x.x.md [hunk 2]
```diff
@@ -42,6 +42,14 @@ Action plugins / Modules:
 
 See the [porting guide](../porting-guides/5.x.x.md#removal-of-ansible-plugins) for details.
 
+### Removal of Ansible roles
+
+The following deprecated ansible role has been removed from the `arista.avd` Ansible collection in v5.0.0.
```
Comment (carlbuchmann) [inline ansible_collections/arista/avd/docs/release-notes/5.x.x.md]:
```suggestion
The following deprecated Ansible role has been removed from the `arista.avd` Ansible collection in v5.0.0.
```

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Review by alexeygorbunov — APPROVED** [review]
LGTM.
Also confirmed with molecule's `cv_deploy` scenario

File: ansible_collections/arista/avd/meta/runtime.yml [hunk 3]
```diff
@@ -69,7 +67,6 @@ plugin_routing:
         warning_text: |-
           The preview module 'arista.avd.deploy_to_cv' has been released under the new name 'arista.avd.cv_workflow'.
           The module will be redirected automatically until AVD version 5.0.0 after which it will be removed.
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/meta/runtime.yml]:
```suggestion
```

**Review by ClausHolbechArista — CHANGES_REQUESTED** [review]

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**